devkit/Dockerfile

```diff
@@ -76,6 +79,8 @@ RUN           curl -LO https://get.helm.sh/helm-v${arg_helm_ver}-linux-amd64.tar
 &&            rm helm-v${arg_helm_ver}-linux-amd64.tar.gz \
 &&            rm -rf linux-amd64
 
+RUN           ln --symbolic /usr/bin/python3 /usr/bin/python
```

Comment:
this is not a good solution.
it won't break anything right?
what will happen to the original file(original python)?

Reply:
It creates a soft symbolic link to source code. Soft link only point to original file, if we decide to delete it, the original files will not be affected.

Reply:
I telling you about other tools which depends on original python.